From aafa5fab74ab2583ce73ff5006259daab8526daf Mon Sep 17 00:00:00 2001
From: Max Horn <max@quendi.de>
Date: Mon, 6 Sep 2021 13:44:13 +0200
Subject: [PATCH] Replace u_int32_t -> uint32_t

This fixes a compile error for me -- uint32_t is a standard type,
u_int32_t is not
---
 YAPB++/source/library/sort_events.hpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/YAPB++/source/library/sort_events.hpp b/YAPB++/source/library/sort_events.hpp
index 14aedc2..e42f301 100644
--- a/YAPB++/source/library/sort_events.hpp
+++ b/YAPB++/source/library/sort_events.hpp
@@ -4,7 +4,7 @@
 #include "library/vec1.hpp"
 #include "promotable_list.hpp"
 
-typedef u_int32_t HashType;
+typedef uint32_t HashType;
 
 struct HashStart
 {
-- 
2.31.0

